libraries/tuttle/src/tuttle/host/memory/MemoryCache.cpp

```diff
@@ -47,6 +47,17 @@ CACHE_ELEMENT MemoryCache::get( const std::size_t& i ) const
 	return itr->second;
 }
 
+CACHE_ELEMENT MemoryCache::getUnusedWithSize( const std::size_t size ) const
+{
+	boost::mutex::scoped_lock lockerMap( _mutexMap );
+	for( MAP::const_iterator itr = _map.begin(); itr != _map.end(); ++itr )
+	{
+		if( size <= itr->second->getMemorySize() )
+			return itr->second;
```

Comment:
Need to find the **unused** elements:
`if( it->second->getReferenceCount( ofx::imageEffect::OfxhImage::eReferenceOwnerHost ) <= 1 )`
See `MemoryCache::clearUnused()`

Maybe better to create a function:
```
/**
 * @brief Check if the cache element is kept in the cache, but is not required by someone else.
 */
bool isUnused( const CACHE_ELEMENT& cacheElement )
{
    return cacheElement->getReferenceCount( ofx::imageEffect::OfxhImage::eReferenceOwnerHost ) <= 1;
}
```